src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,32 +72,28 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Linter

Reply:
It was already there

Reply:
@timoxa2t I know, but you need to fix it